.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You can remove all release-please work from this PR, since I am working on it already and it will be required to configured differently, such that each helm chart has an individual release process.